app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,10 @@
+<% provide(:title, "About") %>
```

Comment:
Mấy file này có dùng ko, ko thì xóa đi, dùng thì I18n cho toàn bộ text nhé, check các file html khác hì